pkg/plugins/shell/main.go

```diff
@@ -18,3 +18,12 @@ func New(spec ShellSpec) (*Shell, error) {
 		spec:     spec,
 	}, nil
 }
+
+func (s *Shell) customCommand(source string) string {
```

Comment:
```suggestion
// Append the source as last argument if not empty. 
func (s *Shell) appendSource(source string) string {
```

customCommand doesn't mean much.